code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -220,7 +220,7 @@
 	name = "stool"
 	desc = "Apply butt."
 	icon_state = "stool"
-	can_buckle = FALSE
+	can_buckle = TRUE
```

Comment:
```suggestion
```
just get rid of the line so it defaults to chair behaviour, no need to set it to TRUE specifically